gatsby/pages.rst

```diff
@@ -1,7 +1,11 @@
 Pages
 =====
 
-Pages can be considered similar to HTML documents in a website, but here, internally they are React components instead with way more functionality and can even be dynamically generated.
+The core part of a website are pages. Every site has at least one HTML page (for example a single page application or a landing page).
+
+GatsbyJS is a static site generator, so it has the concept of pages itself. They only difference is that they are not standard HTML documents, but internally they are React components that will be converted in static HTML on build time.
+
+ReactJS is a good choice because it allows to add more functionalities to the page and can also be dynamically generated.
 
 If we see the file structure of our `hello-world` project, we can see that there is a `pages` folder with some JavaScript files:
```

Comment:
where?

Reply:
Everywhere that single backticks are used AND you want the text to render as inline code instead of italics. Typically file names, file paths, and anything code-like should use double backticks.

Markdown uses single backticks for inline code, whereas reST uses double backticks.

Reply:
s/functionalities/functionality